lib/train/transports/ssh_connection.rb

```diff
@@ -230,7 +230,7 @@ def to_s
 
     class OS < OSCommon
       def initialize(backend)
-        super(backend, { family: 'unix' })
```

Comment:
I just thinking if it is not easier to count openvms as a unix-like system as well? Not sure about that. This would allow us to just use 

```
return true if pf =='unix' && detect_openvms 
```